draft-ietf-sipcore-sip-push.xml

```diff
@@ -207,14 +207,13 @@ align="center"><artwork>
     </section>
 
     <section anchor="section.ua" title="SIP User Agent (UA) Behavior">
+    <section anchor="section.ua.req" title="Request Push Notifications from Network">
       <t>
         Once the SIP UA has registered with the PNS, has received the PRID (using the protocol 
-        and procedures associated with the PNS), and when the UA wants to receive push notifications, 
-        the UA MUST send a SIP REGISTER request using normal SIP procedures. The UA MUST include the 
-        following SIP URI parameters in the SIP Contact header field URI of the REGISTER request: pn-provider, 
-        pn-prid and pn-param (if required for the specific PNS). The pn-provider URI parameter identifies the 
-        PNS, the pn-prid URI parameter contains the PRID value and the pn-param URI parameter contains additional 
-        PNS-specific information.
+        and procedures associated with the PNS), the UA MUST include the following SIP URI parameters in 
+        the SIP Contact header field URI of the REGISTER request: pn-provider, pn-prid and pn-param (if required for the specific PNS). 
+        The pn-provider URI parameter identifies the PNS, the pn-prid URI parameter contains the PRID value and the pn-param URI parameter 
```

Comment:
The pn-provider URI parameter identifies the *type of* PNS

Reply:
I guess I could fix it, but "identifies" terminology is used throughout the document.

Reply:
Then define how the PNS type is identified somewhere common.

Reply:
I think it is easier to simply modify each instance to "type of PNS".